includes/wf_crm_webform_postprocess.inc

```diff
@@ -1803,6 +1811,7 @@ class wf_crm_webform_postprocess extends wf_crm_webform_base {
 
     // Run the Transaction - and Create the Contribution Record - relay Recurring Series information in addition to the already existing Params [and re-key where needed]; at times two keys are required
     $contributionParams['total_amount'] = $contributionFirstAmount;
+    $contributionParams['tax_amount'] = $salesTaxFirstAmount;
```

Comment:
The problem with this code at the moment is that you can easily enter installments=0 via the UI but that causes all the "amount" variables to be unset and throws a fatal exception in CiviCRM because of a DB error - ie. you can't call ContributionRecur/Contribution.create without an amount set.